ui/app/AppLayouts/Onboarding/views/LoginView.qml

```diff
@@ -271,6 +271,8 @@ Item {
                         colorId: model.colorId
                         colorHash: model.colorHash
                         keycardCreatedAccount: model.keycardCreatedAccount
+                        visible: root.startupStore.selectedLoginAccount.keyUid !== model.keyUid
```

Comment:
@endulab the changes you did do the thing, but I see this kind of doing things like masking, can we do it in a more appropriate way (I am aware that we have lot of places where we are doing it this way, but if you agree let's stop that)? Some of the ways are:
- use delegate model
- use filter model
- build correct model from the `onAboutToShow` slot (this is maybe the best approach as it does the thing when it is needed)
- you can even prepare list of accounts each time one select an account

Reply:
@saledjenic from your ideas I followed filter model way, since it is more declarative. I did not want to recreate the model every time or refill the menu with actions. I had to replace Repeater with ListView because of some problems with filtering. Please re-review.

Reply:
@endulab thank you, looks much better now.